apps/pwa/index.html

```diff
@@ -16,7 +16,7 @@
   </head>
   <body>
     <noscript> You need to enable JavaScript to run this app. </noscript>
-    <div id="root"></div>
+    <div id="root">hii</div>
```

Comment:
remove kar dena unwanted chije, bhool jayemge baad mai

Reply:
Fallback page me ye wala file hai, to ye blank page aa rha hai. esa lagta hai ki koi response nahi aya and app is not workign. hii likha hai taki pata chale ki app is working and giving a fallback page

Reply:
hii ke badle fallback page likh deta hoon